File: 1026/yeolyi.js
const fs = require('fs')
const input = fs.readFileSync('/dev/stdin').toString().split('\n')

const arr1 = input[1].split(' ').map(Number).sort((a, b) => a-b)
const arr2 = input[2].split(' ').map(Number).sort((a, b) => b-a)

console.log(arr1.map((e, i) => e*arr2[i]).reduce((a, b) => a+b));

File: 1026/yeolyi.py
from functools import reduce

input()
arr1 = sorted(list(map(int, input().split())))
arr2 = sorted(list(map(int, input().split())), reverse=True)

multiplied = map(lambda x, y: x*y, arr1, arr2)
added = reduce(lambda a, b: a+b, multiplied, 0)
print(added)
